brainframe/cli/frozen_utils.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from pathlib import Path
+
+
+def is_frozen() -> bool:
+    """
+    :return: True if the application is running as a frozen executable
+    """
+    return getattr(sys, "frozen", False)
+
+
+RELATIVE_TRANSLATIONS_PATH = Path("brainframe/cli/translations")
+RELATIVE_DEFAULTS_FILE_PATH = Path("brainframe/cli/defaults.yaml")
+
+
+class ResourceNotFoundError(Exception):
+    pass
+
+
+def defaults_file_path() -> Path:
+    """
+    :return: A path to the included defaults file
+    """
+    if is_frozen():
+        return getattr(sys, "_MEIPASS") / RELATIVE_DEFAULTS_FILE_PATH
```

Comment:
1. `None / RELATIVE_DEFAULTS_FILE_PATH` won't work
2. Are you sure `sys._MEIPASS` is even a `Path`?

Reply:
This applies to `def translations_path()` below as well

Reply:
`getattr(sys, "_MEIPASS")` should never be `None` if the executable is frozen. It is a string, but using the `/` operator with a string and a `Path` works fine.

Reply:
```pycon
>>> from pathlib import Path
>>> "sds" / Path("sd")
PosixPath('sds/sd')
```
huh TIL. I didn't realize `Path` implemented `__rdiv__`

> `getattr(sys, "_MEIPASS")` should never be None

I was thinking `getattr` was like `dict.get` for some reason. Why use `getattr` instead of just `sys._MEIPASS`?

Reply:
Using `getattr` allows us to side-step PyCharm and MyPy's checks, which understandably don't think `_MEIPASS` exists.